common/src/test/java/com/revenuecat/purchases/common/offlineentitlements/OfflineCustomerInfoCalculatorTest.kt

```diff
@@ -49,7 +53,7 @@ class OfflineCustomerInfoCalculatorTest {
         }
         testDateProvider = object : DateProvider {
             override val now: Date
-                get() = dateInThePast
+                get() = requestDate
```

Comment:
This was wrong, we were mocking the request date to a date in the past instead of in the present, making the request date very close to the expires date of one of the products.